pinot-core/src/main/java/com/linkedin/pinot/core/data/extractors/PlainFieldExtractor.java

```diff
@@ -210,15 +209,6 @@ public GenericRow transform(GenericRow row, GenericRow destinationRow) {
             hasError = true;
             _errorCount.put(column, _errorCount.get(column) + 1);
           }
-        } else if (value instanceof ByteBuffer) { // TODO: Need better handle for ByteBuffers.
```

Comment:
why did we remove this?

Reply:
This is moved to transform layer in AvroUtils.